lib/internal/debugger/inspect.js

```diff
@@ -354,12 +351,12 @@ function startInspect(argv = ArrayPrototypeSlice(process.argv, 2),
 
   function handleUnexpectedError(e) {
     if (e.code !== 'ERR_DEBUGGER_STARTUP_ERROR') {
-      console.error('There was an internal error in Node.js. ' +
-                    'Please report this bug.');
-      console.error(e.message);
-      console.error(e.stack);
```

Comment:
There would be a difference if the inspect client listens to `Runtime.consoleAPICalled`, but I don't think it does currently.